src/LiveComponent/src/Attribute/AsLiveComponent.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\UX\LiveComponent\Attribute;
 
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
```

Comment:
Not sure we should couple UrlGeneratorInterface and this attribute.

Reply:
We can invent or own constants and map them over, but the component already bound with the routing component.

Reply:
I agree - we're already bound to routing - so I'm ok with it

Reply:
Allright then